contents/articles/roll-call-brett/index.md

```diff
@@ -0,0 +1,79 @@
+
+Since we are still a small team, we all get to wear many hats and get our hands dirty with every aspect of our platform. Some days I am either writing backend Python or Node.js, writing frontend HTML/CSS/JS or staring at either our [AWS][] management console or [Datadog][] dashboards.
+
+[AWS]: https://aws.amazon.com/
+[Datadog]: http://www.datadoghq.com/
+
+[Underdog.io]: https://underdog.io
+
+#### What hardware do you use?
+
+I use a 21" iMac and a WASD [CODE keyboard][] to help ensure the entire office knows when I am typing something.
+
+[CODE keyboard]: https://codekeyboards.com/
+
+#### What software do you use?
+
+I spend a good majority of my day in the terminal, so for productivity there I use [iterm2][] as a terminal emulator, [oh-my-zsh][] to provide zsh plugins and themes, [homebrew][] for installing third party tools and I use [tmux][] to organize my terminal workspace.
+
+Since I am a software engineer I have to spend a lot of time writing code, for that I use [emacs][] along with some fun packages like [elpy][] to help auto-complete and lint my code while I am writing it.
+
+When I am not writing code, I am usually running it. A few python packages that I do not think I could live without are the [bpython][] interpreter, [pdb++][] and [virtualenvwrapper][].
+
+Lastly, when I am not writing or running code it is usually spent sending fellow team members appropriate emojis and gifs based on the context of some other conversation. For communication at work we use [slack][].
+
+[oh-my-zsh]: https://github.com/robbyrussell/oh-my-zsh
+[homebrew]: http://brew.sh/
+[iterm2]: https://www.iterm2.com/
+[tmux]: http://tmux.github.io/
+[emacs]: https://www.gnu.org/software/emacs/
+[elpy]: https://github.com/jorgenschaefer/elpy
+[bpython]: http://bpython-interpreter.org/
+[pdb++]: https://pypi.python.org/pypi/pdbpp/
+[virtualenvwrapper]: https://virtualenvwrapper.readthedocs.org/en/latest/
+[slack]: http://slack.com/
+
+#### Out of all the tools you use, what is your favorite?
+
+This is a really hard question to answer. I do not think I have any one particular tool that I like more than the others. I think instead I really like my setup as a whole since it is what I have continually refined to help me be productive while at work.
+
+You can check out my [dotfiles][] if you are interested in seeing my full setup.
+
+[dotfiles]: https://github.com/brettlangdon/dotfiles
+
+#### Outside of work, what's your favorite hobby?
+
+Aside from ordering Seamless and binge watching Netflix, I rather enjoy taking nice long walks around the park as well I am an avid chef/baker and always try to be in my kitchen whenever I can (even in the summer!).
+
+### Underdog.io bonus question
+#### What's your favorite interview question?
+
+A question which I heard recently from a friend which I really rather like is "If you could redesign the process for washing and drying clothes, what would your solution look like?"
```

Comment:
Neat question. I had tunnel vision for replying with an engineering focused question whereas this expands to designers.